pkg/virt-launcher/virtwrap/device/hostdevice/sriov/pcipool-network-status.go

```diff
@@ -0,0 +1,77 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2022 Red Hat, Inc.
+ *
+ */
+
+package sriov
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"kubevirt.io/kubevirt/pkg/network/sriov"
+)
+
+type PCIAddressWithNetworkStatusPool struct {
+	networkPciMap map[string]string
+}
+
+// NewPCIAddressPoolWithNetworkStatus creates a PCI address pool based on the networkPciMapPath volume
+func NewPCIAddressPoolWithNetworkStatus(networkPciMapPath string) (*PCIAddressWithNetworkStatusPool, error) {
+	pool := &PCIAddressWithNetworkStatusPool{}
+
+	networkPciMap, err := pool.loadVirtMetadataVolume(networkPciMapPath)
+	if err != nil {
+		return nil, err
+	}
+
+	if networkPciMap == nil {
+		return nil, fmt.Errorf("network-pci map not present")
+	}
+	pool.networkPciMap = networkPciMap
+	return pool, nil
+}
+
+func (p *PCIAddressWithNetworkStatusPool) loadVirtMetadataVolume(networkPciMapPath string) (map[string]string, error) {
```

Comment:
This should be renamed to reflect the new annotation name.

Reply:
Done